src/Hosting/Hosting/src/Microsoft.AspNetCore.Hosting.csproj

```diff
@@ -22,6 +22,7 @@
     <Reference Include="Microsoft.Extensions.Configuration" />
     <Reference Include="Microsoft.Extensions.DependencyInjection" />
     <Reference Include="Microsoft.Extensions.FileProviders.Physical" />
+    <Reference Include="Microsoft.Extensions.Hosting" />
```

Comment:
What’s this for?

Reply:
The UseDefaultServiceProvider extension is in there.

Reply:
Ugh, I don’t like those dependencies here just for that method.

Reply:
I can hoist the code...

Reply:
Yea, hoist it